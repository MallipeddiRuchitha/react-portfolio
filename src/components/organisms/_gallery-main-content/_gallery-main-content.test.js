import React from 'react';
import { shallow ,mount} from 'enzyme';
import GalleryMain from './_gallery-main-content';
import { makeStyles } from "@material-ui/core/styles";

describe("GalleryMain component",() =>{
    test("Matches the snapshot",() => {
        const galleryMain= shallow(<GalleryMain/>);
        expect(galleryMain).toMatchSnapshot();

    })
    test("GalleryMain is rendered",() => {
        const galleryMain= shallow(<GalleryMain/>);
        expect(galleryMain.find("#galleryMain").exists()).toEqual(true);
        
    });
   
   
   

});