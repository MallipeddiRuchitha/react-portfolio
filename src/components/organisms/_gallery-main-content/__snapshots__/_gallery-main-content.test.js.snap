// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryMain component Matches the snapshot 1`] = `
<div
  className="makeStyles-styleGalleryMainContent-1"
  id="galleryMain"
>
  <div
    className="makeStyles-styleGrid-2"
  >
    <Image
      image="https://www.birmingham.ac.uk/Images/News/Forest-900.jpg"
      key="1"
    />
    <Image
      image="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQIWGOJcO5wb_qh4welWgFUgsZPHXMetI-iVg&usqp=CAU"
      key="2"
    />
    <Image
      image="https://miro.medium.com/max/8576/0*zBu6EBAwjXXXHz-z"
      key="3"
    />
    <Image
      image="https://api.timeforkids.com/wp-content/uploads/2019/09/final-cover-forest.jpg"
      key="4"
    />
    <Image
      image="https://www.orfonline.org/wp-content/uploads/2020/03/India-forests-1280x720.jpg"
      key="5"
    />
    <Image
      image="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQfbbQr8RYShw7zga6H2q9kTF78VpUMq22VGA&usqp=CAU"
      key="6"
    />
  </div>
</div>
`;
